packages/webapp/src/components/Task/PureTaskCrops/index.jsx

```diff
@@ -215,7 +215,9 @@ const PureTaskCrops = ({
   const disabled = isRequired && !selectedManagementPlanIds?.length;
 
   useEffect(() => {
-    defaultManagementPlanId && setSelectedManagementPlanIds([parseInt(defaultManagementPlanId)]);
+    if (defaultManagementPlanId && managementPlanIds.includes(+defaultManagementPlanId)) {
```

Comment:
Is the + a typo?

Reply:
No, `managementPlanIds` was an array of numbers and `defaultManagementPlanId` was string, so I converted it to number :)